courses/intro-to-ada/chapters/generics.rst

```diff
@@ -408,20 +504,21 @@ A typical example of an ADT is a stack:
 In this example, we first create a generic stack package (:ada:`Stacks`)
 and then instantiate it to create a stack of 10 integer values.
```

Comment:
I know you didn't change this, but the text shouldn't the text really read "create a stack of up to 10 integer values"?